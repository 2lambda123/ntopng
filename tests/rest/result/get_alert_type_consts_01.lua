{"rc":0,"rc_str":"OK","rsp":{"alert_am_threshold_cross":{"alert_key":30},"alert_blacklisted_country":{"alert_key":1},"alert_broadcast_domain_too_large":{"alert_key":2},"alert_connection_issues":{"alert_key":56},"alert_device_connection":{"alert_key":3},"alert_device_disconnection":{"alert_key":4},"alert_device_protocol_not_allowed":{"alert_key":5},"alert_dropped_alerts":{"alert_key":6},"alert_flow_blacklisted":{"alert_key":8},"alert_flow_blocked":{"alert_key":9},"alert_flow_misbehaviour":{"alert_key":10},"alert_flows_flood":{"alert_key":11},"alert_ghost_network":{"alert_key":12},"alert_host_log":{"alert_key":59},"alert_host_pool_connection":{"alert_key":13},"alert_host_pool_disconnection":{"alert_key":14},"alert_influxdb_error":{"alert_key":16},"alert_influxdb_export_failure":{"alert_key":17},"alert_internals":{"alert_key":18},"alert_ip_outsite_dhcp_range":{"alert_key":19},"alert_known_proto_on_non_std_port":{"alert_key":58},"alert_list_download_failed":{"alert_key":20},"alert_login_failed":{"alert_key":21},"alert_mac_ip_association_change":{"alert_key":22},"alert_malicious_signature":{"alert_key":23},"alert_misconfigured_app":{"alert_key":25},"alert_new_device":{"alert_key":26},"alert_nfq_flushed":{"alert_key":27},"alert_none":{"alert_key":28},"alert_periodic_activity_not_executed":{"alert_key":29},"alert_port_duplexstatus_change":{"alert_key":31},"alert_port_errors":{"alert_key":32},"alert_port_load_threshold_exceeded":{"alert_key":33},"alert_port_mac_changed":{"alert_key":34},"alert_port_status_change":{"alert_key":35},"alert_potentially_dangerous_protocol":{"alert_key":36},"alert_process_notification":{"alert_key":37},"alert_quota_exceeded":{"alert_key":38},"alert_remote_to_remote":{"alert_key":39},"alert_request_reply_ratio":{"alert_key":40},"alert_slow_periodic_activity":{"alert_key":41},"alert_slow_purge":{"alert_key":42},"alert_snmp_device_reset":{"alert_key":43},"alert_snmp_topology_changed":{"alert_key":44},"alert_suspicious_activity":{"alert_key":45},"alert_suspicious_file_transfer":{"alert_key":57},"alert_tcp_syn_flood":{"alert_key":46},"alert_tcp_syn_scan":{"alert_key":47},"alert_threshold_cross":{"alert_key":49},"alert_too_many_drops":{"alert_key":50},"alert_udp_unidirectional":{"alert_key":51},"alert_user_activity":{"alert_key":53},"alert_user_script_calls_drops":{"alert_key":54},"alert_web_mining":{"alert_key":55},"external_alert":{"alert_key":7}}}
