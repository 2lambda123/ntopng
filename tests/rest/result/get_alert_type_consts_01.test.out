{"rc":0,"rc_str":"OK","rsp":[{"key":28,"type":"alert_none"},{"key":20,"type":"alert_list_download_failed"},{"key":12,"type":"alert_ghost_network"},{"key":32,"type":"alert_port_errors"},{"key":56,"type":"alert_connection_issues"},{"key":3,"type":"alert_device_connection"},{"key":11,"type":"alert_flows_flood"},{"key":21,"type":"alert_login_failed"},{"key":7,"type":"external_alert"},{"key":18,"type":"alert_internals"},{"key":36,"type":"alert_potentially_dangerous_protocol"},{"key":51,"type":"alert_udp_unidirectional"},{"key":19,"type":"alert_ip_outsite_dhcp_range"},{"key":5,"type":"alert_device_protocol_not_allowed"},{"key":34,"type":"alert_port_mac_changed"},{"key":13,"type":"alert_host_pool_connection"},{"key":2,"type":"alert_broadcast_domain_too_large"},{"key":16,"type":"alert_influxdb_error"},{"key":47,"type":"alert_tcp_syn_scan"},{"key":44,"type":"alert_snmp_topology_changed"},{"key":27,"type":"alert_nfq_flushed"},{"key":10,"type":"alert_flow_misbehaviour"},{"key":39,"type":"alert_remote_to_remote"},{"key":1,"type":"alert_blacklisted_country"},{"key":53,"type":"alert_user_activity"},{"key":6,"type":"alert_dropped_alerts"},{"key":30,"type":"alert_am_threshold_cross"},{"key":9,"type":"alert_flow_blocked"},{"key":46,"type":"alert_tcp_syn_flood"},{"key":42,"type":"alert_slow_purge"},{"key":58,"type":"alert_known_proto_on_non_std_port"},{"key":17,"type":"alert_influxdb_export_failure"},{"key":54,"type":"alert_user_script_calls_drops"},{"key":49,"type":"alert_threshold_cross"},{"key":40,"type":"alert_request_reply_ratio"},{"key":8,"type":"alert_flow_blacklisted"},{"key":33,"type":"alert_port_load_threshold_exceeded"},{"key":59,"type":"alert_host_log"},{"key":4,"type":"alert_device_disconnection"},{"key":23,"type":"alert_malicious_signature"},{"key":43,"type":"alert_snmp_device_reset"},{"key":35,"type":"alert_port_status_change"},{"key":37,"type":"alert_process_notification"},{"key":29,"type":"alert_periodic_activity_not_executed"},{"key":41,"type":"alert_slow_periodic_activity"},{"key":45,"type":"alert_suspicious_activity"},{"key":25,"type":"alert_misconfigured_app"},{"key":50,"type":"alert_too_many_drops"},{"key":14,"type":"alert_host_pool_disconnection"},{"key":26,"type":"alert_new_device"},{"key":38,"type":"alert_quota_exceeded"},{"key":55,"type":"alert_web_mining"},{"key":22,"type":"alert_mac_ip_association_change"},{"key":57,"type":"alert_suspicious_file_transfer"},{"key":31,"type":"alert_port_duplexstatus_change"}]}
